### PR TITLE
mysql-connector-odbc

### PR DESCRIPTION
**Pull request by kalling**
mysql-connector-odbc package

Note: Building from source always resulted in a library that was only usable on the machine it was built on, so prebuilt packages are downloaded.

Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)

- [X] Source is from official source
- [X] Package does not vend other packages
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by kalling** [comment]
Confirming I am willing to be listed in the maintainer section.

File: recipes/mysql-connector-odbc/build.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+cp ./lib/*.so ${PREFIX}/lib/.
```
Comment (scopatz) [inline recipes/mysql-connector-odbc/build.sh]: These libraries should be rebuilt, not copied.  Is the source code available for them?

Comment (kalling) [reply recipes/mysql-connector-odbc/build.sh]: @scopatz. Yes, but each time I’ve tried building them it at best only worked on the machine it was built on

Comment (scopatz) [reply recipes/mysql-connector-odbc/build.sh]: That should not be the case for conda-forge since we build against a common & consistent set of ABIs

**Pull request closed without merging**